web/src/main/webResources/WEB-INF/data/config/index/records.json

```
@@ -1990,7 +1990,18 @@
         "type": "boolean"
       },
       "indexingErrorMsg": {
-        "type": "keyword"
+        "type": "object",
+        "properties": {
+          "string": {
+            "type": "text"
```

Can be `keyword` because it is a key and we don't run full text search on it.

```suggestion
            "type": "keyword"
```

it will also be used as a facet on admin console > status